src/AVPRIndex/Domain.fs

```diff
@@ -39,6 +39,21 @@ module Domain =
                 )
             | _ -> false
 
+        static member create (
+            fullName: string,
+            email: string,
+            affiliation: string,
```

Comment:
email, affiliation, and affiliation link should be options

Reply:
done